ElliotForWater/efw-webapp
# chore: Update npm modules and recreate package-lock.json

## Description
Re-creating package-lock.json because of previous PR used npm 7 and change many packages versions.
We have npm 6 on our production server, so we will keep this one till we switch everywhere.
This PR is to downgrade to npm 6 and to update some modules with older patches.

#### Impacted Areas in Application
No area in particular.


#### Steps to Test or Reproduce
run the app in dev and check that everything looks like before.

------------------------------------------------
## PR Checklist:
